src/jobs/ega/services/permissions.ts

```diff
@@ -199,16 +292,76 @@ export const processPermissionsForDataset = async (
     for await (const requests of chunkedRevokeRequests) {
       const revokeResponse = await client.revokePermissions(requests);
       if (isSuccess(revokeResponse)) {
-        logger.info(
+        logger.debug(
```

Comment:
This is info logging. We are recording an action we took that changed state and we may want to find a record of in the future.